app/models/pin.rb

```diff
@@ -1,7 +1,14 @@
 class Pin < ApplicationRecord
+  has_many :photos
+
   def update_total_photos
     pin = Pin.find_by_id(params[:id])
     pin.total_photos = pin.total_photos + 1;
```

Comment:
please consider of using counter_cache http://yerb.net/blog/2014/03/13/three-easy-steps-to-using-counter-caches-in-rails/